pkg/webui/console/store/middleware/logics/applications.js

```diff
@@ -31,6 +31,28 @@ const getApplicationLogic = createRequestLogic({
   },
 })
 
+const updateApplicationLogic = createRequestLogic({
+  type: applications.UPDATE_APP,
+  async process ({ action }) {
+    const { id, patch } = action.payload
+
+    const result = await api.application.update(id, patch)
+
+    return { ...result, ...patch }
```

Comment:
any reasons for this?

Reply:
From the request-response, we know 100% that it matches the new data. I could see cases where values might be in some way marshaled or otherwise modified, resulting in a discrepancy between store value and actual value.